tests/AssertionsTest.php

```diff
@@ -32,20 +32,26 @@ protected function setUp(): void
     public function testDomCrawlerAssertions(): void
     {
         self::createPantherClient()->request('GET', '/basic.html');
-        $this->assertSelectorExists('.p-1');
```

Comment:
These changes should be reverted for consistency with the rest of the Symfony codebase.